pkg/mutation/gate.go

```diff
@@ -0,0 +1,24 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mutation
+
+import (
+	"flag"
+)
+
+// MutationEnabled indicates if the mutation feature is enabled
+var MutationEnabled *bool
+
+func init() {
+	MutationEnabled = flag.Bool("enable-mutation", false, "Enable the mutation feature")
```

Comment:
We should note that mutation is in alpha, like we do for events:

`emitAuditEvents           = flag.Bool("emit-audit-events", false, "(alpha) emit Kubernetes events in gatekeeper namespace with detailed info for each violation from an audit")`